### ISSUE TITLE
The anonymous user doesn't work properly

### ISSUE THREAD

**Issue by steinbachr** [MeanPug/folding-together#29]
currently, when someone makes an anonymous donation through the Anonymous user, the balance of the associated anonymous record is incremented appropriately, however because we don't update the stack ID to `empty` upon completion and subsequent termination, a new stack is never launched.

The proposed fix is to update stack ID to `empty` upon termination of FAH instances, which we should probably be doing anyway

**Comment by atkinsonm** [comment]
Yes. This was an oversight on my part. On terminate, we should update the DDB table StackId column to an empty string. We can do this through the Step Function
